lavalink/PlayerManager.py

```diff
@@ -109,6 +123,29 @@ def add(self, requester: int, track: dict):
             await self._lavalink.ws.send(op='play', guildId=self.guild_id, track=track.track)
             await self._lavalink.dispatch_event(TrackStartEvent(self, track))
 
+    async def play_now(self, requester: int, track: dict):
+        """ Add track and play it. """
+        self.add_next(requester, track)
+        await self.play()
+
+    async def play_from_queue(self, index: int):
+        """ Plays the track from a specific index in the queue (moves it to first position). """
+        track = self.queue.pop(min(index, len(self.queue) - 1))
+        self.queue.insert(0, track)
+        await self.play()
+
+    async def play_at(self, index: int):

```

Comment:
Perhaps add an `index` parameter to the play function to prevent the need for queue to be modified and also remove the need for this function